### ISSUE TITLE
KDE: secondary keyboard lay-out not followed when pressing CTRL-SHIFT-V

### ISSUE THREAD

**Issue by tneo** [keepassxreboot/keepassxc#7242]
## Overview
When the user has set-up a keyboard layout configuration and using KeepassXC to paste password and user names using CTRL-V the keyboard layout is not followed as set by the layout selection, but it follows the layout as default for the system. 

## Steps to Reproduce
1. Configure an additional keyboard lay-out (for instance DVORAK), enable a key combination to switch between the two.
2. Open Kate for the test.
3. Select your secondary keyboard layout
4. Select KeepassXC and select credentials to be inserted by keepassXC 
5. Press CTRL-V and observe that in Kate the credentials inserted follow your system default lay-out and not the secondary keyboard lay-out.

## Expected Behavior
KeepassXC should follow the selected keyboard lay-out and not the system default. 

## Actual Behavior
System default keyboard lay-out is used, and the user has to switch to the system lay-out before pasting the password combination. This only happens when inserting username and password using CTRL-V. While the secondary lay-out is used and the user wants to copy the password with CTRL-C that works, same for the username with CTRL-B. 

## Context
My laptop follows both DVORAK and QWERTY lay-out. QWERTY is the default, but I use DVORAK only. As I share this laptop with the family I need to software switch between lay-outs.

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - Version 2.6.6
Revision: 9c108b9

Qt 5.12.7
Debugging mode is disabled.

Operating system: openSUSE Leap 15.3
CPU architecture: x86_64
Kernel: linux 5.3.18-59.37-default

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey
- Secret Service Integration

Cryptographic libraries:
- libgcrypt 1.8.2

Windowing System: X11/Wayland (Both)

**Comment by droidmonkey** [comment]
So are you saying that _within Kate_ the credentials are pasted as if they are using the primary (QWERTY) layout? It is hard to follow your reproduction steps, you are missing a couple key points at the end.

Secondly, are you referring to Copy/Paste or Auto-Type? Copy Paste has nothing to do with keyboard layouts.

**Comment by tneo** [comment]
Sorry I thought it was all covered. And I was not correct with the keyboard combination keys. I'm referring to auto-type.

5. When you press the auto-type combination (CTRL-SHIFT-V) to insert the credentials from KeepassXC. You can observe within Kate the credentials are pasted using QWERTY (system default) and **NOT** the currently active DVORAK lay-out. _(I'm using Kate as it is a visible example that auto-type is not properly working. Same behaviour is seen in browsers.)_

**Comment by droidmonkey** [comment]
In which case your issue is solved by this fix and will be out with 2.7.0. https://github.com/keepassxreboot/keepassxc/pull/6247
